### PR TITLE
HRI-63-Adding the initial JSON for the renLearn Star score

### PR DESCRIPTION
**Pull request by VibinD**

### CODE REVIEW

**Comment by VibinD** [comment]
Created for review and once it approved needs to get merged

File: src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "scoreGiven": 500,
+    "scoredBy": "https://www.hmhco.com/PartnerService",
+    "comment": "The Assessment score from RenLearn # c35603f7a3434e2cb74ef14d60e70f42 taken on date/time for student 0f4fedbe-2227-415f-8553-40731a627171",
+    "dateCreated": "2016-11-15T10:56:00.000Z"
+  },
+  "membership":{
+    "id":"urn:uuid:8f4fedbe-2227-415f-8553-40731a627171", // random UUID
+    "type":"Membership",
+    "organization":{
+      "id":"urn:uuid:ab570d90-3791-4048-a5ee-759657ddccef", // random UUID
```
Comment (odowdj) [inline src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json]: this isn't a random id, it must be the school ref id that the student belongs to

File: src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      "type":"Organization",
+      "subOrganizationOf":{
+        "id":"urn:uuid:ea827de8-0ce9-4fe4-a28c-eb388bd9eb92", // random UUID
```
Comment (odowdj) [inline src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json]: this isn't a random id, it must be the district ref id that is associated with the school ref id above it

**Review by odowdj — CHANGES_REQUESTED** [review]

File: src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json (hunk 1 above)
Comment (VibinD) [reply src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json]: Sure. I can add the school ref id for this.

File: src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json (hunk 2 above)
Comment (VibinD) [reply src/test/resources/fixtures/hmh-renlearn/caliperEventStarScoreGraded.json]: Sure. I can add the district ref id for the school which student is associated in.

**Review by odowdj — APPROVED** [review]
lftm

**Comment by VibinD** [comment]
Thanks @odowdj. Can you please merge this also. Looks like I don't have permission to merge this.

**Pull request merged**